c2tc-mobile/components/PanelComponent/ButtonInterface.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Button, StyleSheet, View, Dimensions } from "react-native";
+
+const { width, height } = Dimensions.get("window");
+
+export default class ButtonInterace extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      type: this.props.type
+    };
+  }
+
+  componentDidMount() {}
+
+  getType() {
+    return this.state.type;
+  }
+
+  _onPressLearnMore() {
+    alert("on Press!");
+  }
+
+  render() {
+    return (
+      <Button
+        onPress={this._onPressLearnMore}
+        title={this.props.type}
+        color="#841584"
```

Comment:
Please make this a variable.

Reply:
@choiboy98 please make the changes from both Neeraj and my reviews